std/net/udp.zig

```diff
@@ -0,0 +1,121 @@
+const std = @import("../index.zig");
+const event = std.event;
+const assert = std.debug.assert;
+const os = std.os;
+const mem = std.mem;
+const net = std.net;
+
+pub fn UDP() type {
```

Comment:
if it doesn't take any comptime args, then you can just do `pub const UDP = struct { ...`